### PR TITLE
AUV ingest CSV cleanup

### PR DESCRIPTION
**Pull request by cdobs**
Made the following changes:

CP05MOAS-A6263:
-R1: There were no recovered data for this cruise but an ingest csv should still exist. Created it.
-R5: though it was an aborted deployment with no data, it was deployed with a NB CTD so this ingest csv technically needs that driver

CP05MOAS-A6264:
-R4: For some reason there were 2 lines for CTD data, one with the NB driver and one with the old driver. This was the first deployment with NB for this vehicle. Line with old CTD driver deleted
-R5: though it was an aborted deployment with no data, it was deployed with a NB CTD so this ingest csv technically needs that driver

CP02PMCI:
R2: The data for this deployment were previously considered unrecovered but the raw data were located by J. Lund and are now available - except the recovered host ADCP data because the inductive ADCP comms failed

### CODE REVIEW

File: CP02PMCI/CP02PMCI_R00002_ingest.csv [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/*/A*.DA?,CP02PMCI-WFP01-01-VEL3DK000,recovered_wfp,Available,not recovered
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/*/C*.DA?,CP02PMCI-WFP01-02-DOFSTK000,recovered_wfp,Available,not recovered
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/*/C*.DA?,CP02PMCI-WFP01-03-CTDPFK000,recovered_wfp,Available,not recovered
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/*/E*.DA?,CP02PMCI-WFP01-04-FLORTK000,recovered_wfp,Available,not recovered
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/*/E*.DA?,CP02PMCI-WFP01-05-PARADK000,recovered_wfp,Available,not recovered
```
Comment (pbrickley) [inline CP02PMCI/CP02PMCI_R00002_ingest.csv:11]: Good

File: CP05MOAS-A6263/CP05MOAS-A6263_R00005_ingest.csv [hunk 2]
```diff
@@ -2,7 +2,7 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
-#mi.dataset.driver.ctdav_n.auv.ctdav_n_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.txt,CP05MOAS-A6263-03-CTDAVN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+#mi.dataset.driver.ctdav_nbosi.auv.ctdav_nbosi_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.txt,CP05MOAS-A6263-03-CTDAVN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
 #mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/DATA/*.SUN,CP05MOAS-A6263-04-NUTNRN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
 #mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/ADCP/*.ADC,CP05MOAS-A6263-05-ADCPAN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
 #mi.dataset.driver.parad_n.auv.parad_n_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00005/*.txt,CP05MOAS-A6263-06-PARADN000,recovered_host,Not Expected,mission aborted at 13:36 UTC due to ground fault (GFI); no data
```
Comment (pbrickley) [inline CP05MOAS-A6263/CP05MOAS-A6263_R00005_ingest.csv:8]: Fine, really a placeholder

File: CP05MOAS-A6264/CP05MOAS-A6264_R00005_ingest.csv [hunk 3]
```diff
@@ -3,7 +3,7 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
-#mi.dataset.driver.ctdav_n.auv.ctdav_nbosi_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00005/*.txt,CP05MOAS-A6264-03-CTDAVN000,recovered_host,Not Expeted,mission aborted at 13:36 UTC due to ground fault (GFI); no data
+#mi.dataset.driver.ctdav_nbosi.auv.ctdav_nbosi_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00005/*.txt,CP05MOAS-A6264-03-CTDAVN000,recovered_host,Not Expeted,mission aborted at 13:36 UTC due to ground fault (GFI); no data
 #mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00005/DATA/*.SUN,CP05MOAS-A6264-04-NUTNRN000,recovered_host,Not Expeted,mission aborted at 13:36 UTC due to ground fault (GFI); no data
 #mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00005/ADCP/*.ADC,CP05MOAS-A6264-05-ADCPAN000,recovered_host,Not Expeted,mission aborted at 13:36 UTC due to ground fault (GFI); no data
 #mi.dataset.driver.parad_n.auv.parad_n_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6264/R00005/*.txt,CP05MOAS-A6264-06-PARADN000,recovered_host,Not Expeted,mission aborted at 13:36 UTC due to ground fault (GFI); no data
```
Comment (pbrickley) [inline CP05MOAS-A6264/CP05MOAS-A6264_R00005_ingest.csv:9]: Good, another placeholder

File: CP05MOAS-A6263/CP05MOAS-A6263_R00001_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+#mi.dataset.driver.flort_kn.auv.flort_kn_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00001/*.txt,CP05MOAS-A6263-01-FLORTN000,recovered_host,Not Expected,Test deployment; no data
+#mi.dataset.driver.dosta_ln.auv.dosta_ln_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00001/*.txt,CP05MOAS-A6263-02-DOSTAN000,recovered_host,Not Expected,Test deployment; no data
+#mi.dataset.driver.ctdav_n.auv.ctdav_n_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00001/*.txt,CP05MOAS-A6263-03-CTDAVN000,recovered_host,Not Expected,Test deployment; no data
+#mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00001/DATA/*.SUN,CP05MOAS-A6263-04-NUTNRN000,recovered_host,Not Expected,Test deployment; no data
+#mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00001/ADCP/*.ADC,CP05MOAS-A6263-05-ADCPAN000,recovered_host,Not Expected,Test deployment; no data
+#mi.dataset.driver.parad_n.auv.parad_n_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00001/*.txt,CP05MOAS-A6263-06-PARADN000,recovered_host,Not Expected,Test deployment; no data
```
Comment (pbrickley) [inline CP05MOAS-A6263/CP05MOAS-A6263_R00001_ingest.csv:8]: Good

**Review by pbrickley — APPROVED** [review]

File: CP02PMCI/CP02PMCI_R00002_ingest.csv [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
-#mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/C*.DA?,CP02PMCI-WFP01-03-CTDPFK000,recovered_wfp,Not Expected,not recovered
-#mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/E*.DA?,CP02PMCI-WFP01-04-FLORTK000,recovered_wfp,Not Expected,not recovered
-#mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/E*.DA?,CP02PMCI-WFP01-05-PARADK000,recovered_wfp,Not Expected,not recovered
\ No newline at end of file
+#mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/imm/adcpt/adcpt*.DAT,CP02PMCI-RII01-02-ADCPTG010,recovered_host,Not Expected,ADCP Indcutive comms failed
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/ADCP*/*.000,CP02PMCI-RII01-02-ADCPTG010,recovered_inst,Available,not recovered
```
Comment (snwhite) [inline CP02PMCI/CP02PMCI_R00002_ingest.csv:3]: The Raw Data Repo has 2 files that end with .000  (CP02PMCI-00002_ADCP_20195_RDI_000.000 and _RDI_000.000).  They are the same size, and presumably duplicates.

**Review by snwhite — APPROVED** [review]
Looks good.  I had one question on PMCI ADCP data in the Raw Data Repo.

**Pull request merged**